nomad/local/integration-tests.nomad

```diff
@@ -270,40 +169,73 @@ job "integration-tests" {
         }
       }
     }
-    task "graphql-endpoint-tests" {
-      driver = "docker"
-
-      config {
-        image      = "${var.container_registry}grapl/graphql-endpoint-tests:dev"
-        entrypoint = ["/bin/bash", "-o", "errexit", "-o", "nounset", "-c"]
-        command    = "cd graphql_endpoint_tests; py.test --capture=no -n 1 -m 'integration_test'"
-      }
 
-      env {
-        # aws vars
-        AWS_REGION                  = var.aws_region
-        GRAPL_AWS_ENDPOINT          = local.aws_endpoint
-        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
-        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
+    volume "grapl-root-volume" {
+      # The definition of this `grapl-root-volume` is written as a Nomad agent
+      # config in `start_detach.sh`
+      type      = "host"
+      source    = "grapl-root-volume"
+      read_only = false
+    }
 
-        DEPLOYMENT_NAME = var.deployment_name
-        GRAPL_LOG_LEVEL = local.log_level
+    # This is hacky as hell.
+    task "python-integration-tests" {
+      driver = "exec"
+      user   = var.non_root_user
 
-        GRAPL_SCHEMA_PROPERTIES_TABLE = var.schema_properties_table_name
+      volume_mount {
+        volume      = "grapl-root-volume"
+        destination = "/mnt/grapl-root"
+        read_only   = false
+      }
 
-        # These are placeholders since Ian is replacing the nginx service shortly
-        GRAPL_API_HOST           = "localhost"
-        GRAPL_HTTP_FRONTEND_PORT = "${NOMAD_UPSTREAM_PORT_web-ui}"
-        GRAPL_TEST_USER_NAME     = var.test_user_name
+      config {
+        command = "/bin/bash"
+        args = [
+          "-o", "errexit", "-o", "nounset", "-c",
+          trimspace(<<EOF
+# aws vars
```

Comment:
Notably, `exec` tasks do not support env { }

Reply:
Gosh that's a pain. This is fine for now, but I suspect we'll want a nicer pattern in the future